src/GitHubCreateMergePRs/config.xml

```diff
@@ -12,11 +12,10 @@
     <merge from="release/dev17.0-vs-deps" to="release/dev17.1" />
 
     <!-- Roslyn branches (from non-vs-deps to vs-deps) -->
-    <merge from="release/dev17.1" to="release/dev17.1-vs-deps" />
     <merge from="main" to="main-vs-deps" />
 
     <!-- Roslyn release branches (flowing between releases) -->
-    <merge from="release/dev17.1-vs-deps" to="release/dev17.2" />
+    <merge from="release/dev17.1" to="release/dev17.2" />
     <merge from="release/dev17.2" to="main" />
     <merge from="main" to="release/dev17.3" />
```

Comment:
incidentally, should 17.3 branch contain any vs-deps bits here? Are we hoping to remove -vs-deps again before it's time to start inserting 17.3 anyway?

Reply:
Oh, great point - I think it might be good to set up that flow just in case. Will open a follow-up PR since this one already auto-merged